spec.md

```diff
@@ -1696,23 +1817,25 @@ The following table summarizes the binary arithmetic operations
 available for built-in types:
 
 ```text
-Arithmetic
+Arithmetic (int or float; result has type float unless both operands have type int)
    number + number              # addition
    number - number              # subtraction
    number * number              # multiplication
+   number / number              # real division (result is always a float)
    number // number             # floored division
    number % number              # remainder of floored division
-   number ^ number              # bitwise XOR
-   number & number              # bitwise AND
-   number | number              # bitwise OR
-   number << number             # bitwise left shift
-   number >> number             # bitwise right shift
+
+Bitwise operations:
```

Comment:
Add `(int only)`?

Reply:
Not necessary. The "int or float:" comment above is to clarify patterns such as `number + number`, but here the patterns all say int.